.github/workflows/nightly-te-test.yaml

```
@@ -12,7 +12,7 @@ on:
         type: string
         description: 'JAX-TE image build by NVIDIA/JAX-Toolbox'
         required: true
-        default: 'ghcr.io/nvidia/jax-te:latest'
+        default: 'ghcr.io/nvidia/jax:latest'
```

Same Q about upstream-pax?
```suggestion
        default: 'ghcr.io/nvidia/upstream-pax:latest'
```